app/views/provider/admin/onboarding/wizard/backend_api/new.html.slim

```diff
@@ -0,0 +1,28 @@
+h1 Add your API(s)
+
+ol.explain.fa-ul
+  li.feedback
+    i.icon.fa-li.fa.fa-cube.icon--focus
+    p.text.text--up
+      ' In 3scale each of your APIs is represented by what we call a
+      strong Backend
+      | .
+    p.text.text--up
+      ' A Backend contains the
+      strong> Base URL
+      ' of your API and helps you to keep track of requests through
+      strong> Metrics, Methods and Mapping Rules.
+    p.text.text--up
+      ' Let's add your first Backend (If you don’t have a publicly accessible API yet, please
```

Comment:
"publicly" makes sense for hosted apicast in Saas, but possibly not so much for some other scenarios.

```suggestion
      ' Let's add your first Backend (If you don’t have an accessible API yet, please
```